File: leetcode/Study Plan/744. 寻找比目标字母大的最小字母/3.py
'''
Author: tkzzzzzz6
Date: 2026-01-31 19:44:17
LastEditors: tkzzzzzz6
LastEditTime: 2026-01-31 19:44:25
'''
class Solution:
    def nextGreatestLetter(self, letters: List[str], target: str) -> str:
        i = bisect_right(letters,target)
        return letters[i] if i < len(letters) else letters[0]

File: niuke/daily_problem/26_3-1二进制数1/3.cpp
/*
 * @Author: tkzzzzzz6
 * @Date: 2026-03-02 09:01:57
 * @LastEditors: tkzzzzzz6
 * @LastEditTime: 2026-03-02 09:02:03
 */
#include <iostream>
using namespace std;

int main() {
    long long n;
    cin >> n;
    long long ans = 0;
    while (n > 0) {
        ans += n & 1;
        n >>= 1;
    }
    cout << ans << endl;
}
